helm/coredns-app/templates/deployment-masters.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: apps/v1
```

Comment:
Same as the previous one, just renamed with `-cp` suffix and added toleration and node selector for master nodes